models/public/midasnet/midasnet.md

```diff
@@ -0,0 +1,20 @@
+# midasnet
```

Comment:
Please, add information about I/O, and summary about model. See [here](https://github.com/opencv/open_model_zoo/blob/develop/models/public/alexnet/alexnet.md) for example.

Reply:
readme updated.